docs/src/ref/distributions.md

```diff
@@ -19,3 +17,207 @@ exponential
 laplace
 ```
 
+## Creating New Distributions via the Distributions DSL
+
+In addition to using the above built-in distributions, and to defining subtypes
+of `Gen.Distribution` in plain Julia, you may combine existing distributions to
```

Comment:
You can't use the `@dist` DSL to combine existing distributions, unfortunately -- only one random choice per `@dist` :)

Reply:
Fixed.